profiler/wdt_follow_ajax.rst

```diff
@@ -0,0 +1,40 @@
+.. index::
+    single: Profiling: WDT Auto-update after AJAX Request
+
+How to make the Web Debug Toolbar Auto-upate after AJAX Requests
+================================================================
+
+For single page applications it is often inconvenient to find the profiler
+information for the most recent request. It would be much more convient if it
+the toolbar would show the information from the most recent AJAX request
+instead of your initial page load.
+
+By setting the `Symfony-Debug-Toolbar-Replace` to a value of `1` in the
+AJAX request, the toolbar will be automatically reloaded for the request. The
+header can be set on the response object::
+
+    $response->headers->set('Symfony-Debug-Toolbar-Replace', 1);
+
+Only setting the header durring development
+-------------------------------------------
+
+Ideally this header should only be set durring development and not for
```

Comment:
typo `during`

Reply:
Perhaps:
> As the Web Debug Toolbar should not be used in production, this header should only be set in the development environment. This can be accomplished by creating a `kernel.response` event listener that is only registered in the development environment.